### ISSUE TITLE
Error: Global name 'pptr' already in use

### ISSUE THREAD

**Issue by gsal** [ElmerCSC/elmerfem#9]
Say, I just followed the [git-cloning and building instructions](https://csc.fi/web/elmer/sources-and-compilation) and got an error.  I would appreciate some guidance. 

/home/gsal/Downloads/elmer/elmerfem/fem/src/LoadMod.F90:321.21:

```
        intval = pptr(model)
                 1
```

/home/gsal/Downloads/elmer/elmerfem/fem/src/LoadMod.F90:572.54:

```
                  M, D, S, F, element, nrow, ncol)
                                                  2
```

Error: Global name 'pptr' at (1) is already being used as a SUBROUTINE at (2)
make[3]: **\* [fem/src/CMakeFiles/elmersolver.dir/LoadMod.F90.o] Error 1
make[2]: **\* [fem/src/CMakeFiles/elmersolver.dir/LoadMod.F90.o.provides] Error 2
make[1]: **\* [fem/src/CMakeFiles/elmersolver.dir/all] Error 2
make: **\* [all] Error 2

thx. 

gsal

**Comment by raback** [comment]
Hi, I think your compiler may be too old. Look at similar problems in the forum at http://www,elmerfem.org/forum. Search for "pptr" and you get at least one hit. -Peter

**Comment by mbycklin** [comment]
Issue has been resolved in commit cd03e55. CMake now checks that the compiler used properly supports procedure pointers.
